src/main/java/fi/foyt/foursquare/api/NotificationsParser.java

```diff
@@ -81,6 +81,12 @@ private NotificationsParser() {
               ScoreNotification scoresNotification = (ScoreNotification) JSONFieldParser.parseEntity(ScoreNotification.class, item, skipNonExistingFields);
               result.add(new Notification<ScoreNotification>(notificationType, scoresNotification));
             break;
+            case NotificationTray:
+              // TODO
```

Comment:
I believe so. I made the change a while back. Looks like I hacked it in to prevent FoursquareApiException.